src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -154,19 +156,19 @@ const Contact: React.FC<Props> = ({
                   value={values.tagValue}
                   onChange={(event) => handleChange(event, 'tagValue')}
                 >
-                  {contactTags.map((t) => (
+                  {Object.keys(contactTags).map((key) => (
```

Comment:
Since you seem to be using both the key and the value, I would do
```suggestion
                  {Object.entries(contactTags).map(([key, value]) => (
```
(maybe there are better names for those variables in this specific context)